experimental/bazel-base/Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN apt-get update && \
         git \
         gnupg2 \
         g++ \
+        iproute2 \
```

Comment:
Can we add a comment here to indicate that it's a dependency for mobilityd?

Reply:
added comment